### DISCUSSION TITLE
Can I customize each modi separately ?

### DISCUSSION THREAD

**Discussion by adhyuthn** [davatorium/rofi#1897] [Q&A]
My current rofi setup has 3 modi(s) - windows, drun, find, run.
I want to customize the 'element' widget separately for the find option. Like is there a concept of selector in rofi, like in CSS? So that I can access the element in find with something like find.element{ }

**Comment by DaveDavenport** [comment]
No, this is not supported. 

See https://github.com/davatorium/rofi/issues/1356 for open issue.
